apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx

```diff
@@ -65,6 +65,9 @@ export const BreakpointsSelector = ({
         type="single"
         value={selectedBreakpoint.id}
         onValueChange={(breakpointId: string) => {
+          if (!breakpoints.has(breakpointId)) {
```

Comment:
Proper comment for this:
```suggestion
          // When user clicks on a selected breakpoint, we don't want to deselect it.
          if (breakpoints.has(breakpointId) === false) {
```